modules/ossm-release-notes-istio-deployment-lifecycle.adoc

```diff
@@ -0,0 +1,58 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-support-tables-assembly.adoc
+////
+
+//Post GA: might be worth considering putting tables into their own modules. Might easier to maintain. Warrants further discussion post GA.
```

Comment:
Just wondering if this note can be removed throughout the modules, since it seems you did move them into their own modules?